### PR TITLE
Add cancelOrder and activateOrder mutations v2

### PR DESCRIPTION
**Pull request by sbinlondon**

### CODE REVIEW

File: server/graphql/v2/mutation/OrderMutations.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+import { NotFound, Unauthorized } from '../../errors';
+import { getDecodedId } from '../identifiers';
+import { Order } from '../object/Order';
+
+const orderMutations = {
+  cancelOrder: {
+    type: Order,
+    description: 'Cancel an order',
+    args: {
+      id: {
```
Comment (znarf) [inline server/graphql/v2/mutation/OrderMutations.js:15]: I would use a "ReferenceInput" similar to what we did for account or expense.

File: server/graphql/v2/mutation/OrderMutations.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+          id: decodedId,
+        },
+        include: [
+          { model: models.Subscription },
+          { model: models.Collective, as: 'collective' },
+          { model: models.Collective, as: 'fromCollective' },
+        ],
+      };
+
+      const order = await models.Order.findOne(query);
```
Comment (znarf) [inline server/graphql/v2/mutation/OrderMutations.js:38]: This block is quite long and could be reused across the 2 mutations.

File: server/graphql/v2/mutation/OrderMutations.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  { model: models.Collective, as: 'collective' },
+  { model: models.Collective, as: 'fromCollective' },
+];
+
+const orderMutations = {
+  cancelOrder: {
+    type: Order,
+    description: 'Cancel an order',
+    args: {
+      input: {
```
Comment (kewitz) [inline server/graphql/v2/mutation/OrderMutations.js:22]: This argument is usually named after the object type.
```suggestion
      order: {
```

File: server/graphql/v2/mutation/OrderMutations.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+      });
+
+      return models.Order.findOne(query);
+    },
+  },
+  activateOrder: {
+    type: Order,
+    description: 'Reactivate a cancelled order',
+    args: {
+      input: {
```
Comment (kewitz) [inline server/graphql/v2/mutation/OrderMutations.js:74]: Same here, rename it to `order` for consistency.

File: server/graphql/v2/mutation/OrderMutations.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    type: Order,
+    description: 'Cancel an order',
+    args: {
+      input: {
+        type: new GraphQLNonNull(OrderReferenceInput),
+        description: 'Object matching the Order reference Input (id)',
+      },
+    },
+    async resolve(_, args, req) {
+      const decodedId = getDecodedId(args.input.id);
```
Comment (kewitz) [inline server/graphql/v2/mutation/OrderMutations.js:28]: 💯

File: server/graphql/v2/mutation/OrderMutations.js (hunk 3 above)
Comment (sbinlondon) [reply server/graphql/v2/mutation/OrderMutations.js:22]: Thanks for that! I didn't know. Will update.

**Comment by sbinlondon** [comment]
Idk what's going on with CodeCov 😐 I've rebased onto the latest master.

**Comment by znarf** [comment]
@sbinlondon wait E2E to complete.

**Comment by sbinlondon** [comment]
@znarf Everything passes but Codecov.

**Comment by znarf** [comment]
@sbinlondon we can ignore, not an issue with the PR itself.

**Comment by sbinlondon** [comment]
@znarf Ok to merge then or do you want more done with the code?

**Comment by znarf** [comment]
@sbinlondon We can merge. I might do a couple of edits later.

**Pull request merged**